sandbox/render_blender_demo.py

```
@@ -0,0 +1,119 @@
+        "beauty",
+        command=["/bin/echo", "rendered", "frame", "#IFRAME#"],
+        range="1-%d" % args.frames,
+    ))
+    # Paused so the instant marker frames don't finish and drop the job out of
+    # Monitor Jobs before you can open it; the images exist regardless.
+    outline.cuerun.launch(ol, pause=True, use_pycuerun=False)
+
+    # Poll for visibility rather than a one-shot lookup: _find_job can race
+    # Cuebot right after launch and would skip registerOutputPath.
+    job = _wait_for_jobs([short_name])[0]
+    layer = next(l for l in job.getLayers() if l.name() == "beauty")
```

Ruff (0.15.17)

[error] Ambiguous variable name: `l`

(E741)